content/trailmap/prologue.md

```diff
@@ -0,0 +1,159 @@
+the more adoption the more Dogecoin gains traction as a serious universal currency 
+for people everywhere. 
+
+## Two coins starting with D..
+
+
+<div class="center">
+<img src="/denarius.png" width='50%' style='margin: auto;' alt="Denarius to Dogecoin">
+</div>
+
+There was a time when most of the 'known world' accepted a universal currency, that
+minted by Rome. A **Denarius**, 'about a day's wages' had one thing going for it, you 
+could spend it anywhere in the empire to satisfy your wants or needs, it had **utility**.
+
+A Denarius had no smart contracts, no scriptability, no DAPPs, and yet this coin once 
+controlled the world.
+
+
+We believe **Dogecoin** is on a simmilar trajectory, yet this time it's not the face 
+of the Emperor controlling a centralised currency, but a friendly Doge welcoming 
+all to a decentralised and liberated currency for the people. 
+
+Whilst many crypto projects are trying to be stores of value such as BTC, or 
+engineering platforms such as ETH, we believe that the _innovative step_ has 
+already been made by Sotashi with the invetion of crypto _CURRENCY_.
+
+Dogecoin already has the fundamentals required to achieve these goals by focusing
+on simplicity, utility & reliability, and that's what this Trailmap is focused on.
+
+## Utility through rapid integration.
+
+For the last eight years Dogecoin has closely tracked the aging Bitcoin C++/QT 
+desktop wallet, heroically maintained by the Dogecoin Core dev team. A decade 
+ago when crypto was in it's infancy this was a solid decision, but times change 
+and what was once useful at the foundation of crypto has lagged behind a world
+where most people transact on their phones, and integration with online retailers
+is paramount. 
+
+The Foundation Trailmap provides a path forward to both secure Dogecoin
+through network diversification (paving the way for new, faster Nodes), and
+consolidating what Dogecoin is through a written, provable Dogecoin Standard with 
+comprehensive testing.
+
+Furthermore it presents new avenues for payment providers, retailers, game 
+developers and social platforms (to name a few), to rapidly integrate Dogecoin 
+into their infrastructure with scalable backend solutions such as the 
+[GigaWallet Project](/trailmap/gigawallet/).
+
+Community safety is tackled with a proposed Dogecoin Keyring App & SDK that will 
+let Dogecoin users retain custody of their Dogecoin while integrating with the
+GigaWallet APIs so that mobile apps can transact Doge without holding your keys.
+
+The Foundation are also working with Vitalik (Founder of the Ethereum blockchain)
+on crafting a uniquely _Doge_ proposal for a 'Community Staking' version of PoS 
+that will allow _everyone_, not just the big players to participate in a way that
+rewards them for their contribution to running the network, and at the same time 
+gives back to the whole community through charitable causes.
+
+## Timeline
```

Comment:
does this need a '.' at the end of 'Timeline' for consistency of sub-titles as above?